README.md

```diff
@@ -41,3 +41,5 @@ Alternatively you can provide a custom path to the properties file using `-Dprop
 If your Kafka is secured you will need to enable the security configuration options in your properties file.
 
 To increase the logging level to debug provide a system property at start time: `-Dlog.level=debug`.
+
+To increase the Kafka client logging level to info provide a system property at start time: `-Dlog.level.kafka=info`.
```

Comment:
how come this line states the Kafka client but the previous line doesn't?

Reply:
The log.level is a general level for the whole app, the log.level.kafka just changes the kafka client level